### PR TITLE
Showing expected header value in the error message

### PR DESCRIPTION
**Pull request by netmilk**

### CODE REVIEW

File: src/utils/tv4-to-headers-message.coffee [hunk 1]
```diff
@@ -1,4 +1,6 @@
-module.exports = (message) ->
+caseless = require 'caseless'
+
+module.exports = (message, expectedHeaders) ->
   if message.indexOf("Missing required property:") > 1
```
Comment (kuba-kubula) [inline src/utils/tv4-to-headers-message.coffee]: should here be `> 1`? Not `> -1`?

Comment (netmilk) [inline src/utils/tv4-to-headers-message.coffee]: Probs yes ;)

**Comment by kuba-kubula** [comment]
:100: points for using `caseless`.

**Comment by netmilk** [comment]
🍬

**Pull request merged**